packages/terra-search-field/src/SearchField.jsx

```diff
@@ -11,6 +11,16 @@ const cx = classNames.bind(styles);
 
 const propTypes = {
   /**
+   * A boolean that when true will disable the field
+   */
+  isDisabled: PropTypes.bool,
+
+  /**
+   * A boolean that when true will stretch the field to fill its container
+   */
+  isStretched: PropTypes.bool,
```

Comment:
What do folks think about changing this to `isBlock` to be more consistent with other apis, namely button. Would describing this behavior as block be an abuse of the term?

Reply:
isBlock would make me believe how it will display next to its peers, currently it is display: inline-flex, which makes it atomic to its peers and always displays inline

I have the width set to 100% to match with the select stying; however, setting it to be display: flex allows it to fill its container as well

If changing the display attribute to be flex, then I think isBlock makes sense as the 'flex' style makes it a block-level element

After thinking more on this, I am more in favor of using isBlock and using display: flex and let it size itself next to its peers rather than always be 100% that maybe could cause display issues that I have been experiencing with the collapsible menu view

Reply:
👍